sushibar-backend-kafka/src/main/java/org/virtualsushibar/backend/service/impl/OrderServiceImpl.java

```diff
@@ -0,0 +1,10 @@
+package org.virtualsushibar.backend.service.impl;
+
+import org.springframework.stereotype.Service;
+import org.virtualsushibar.backend.avro.Order;
+
+@Service
```

Comment:
Why an interface ends with impl? And does it make sense to have @Service annotation. I’ll check better later

Reply:
Spring needs this annotation to be autodetcted 
https://www.baeldung.com/spring-service-annotation-placement

Reply:
Tried to use order service from an interface Rather than directly

Reply:
Ah ok never tried. Ok if it works fine

Reply:
found my mistake.. correcting it